mock-backend/src/controllers/productController.js

```diff
@@ -10,15 +12,18 @@ const getAllProducts = (req, res) => {
   const page = validateNumberQueryParam(req.query.page);
   const size = validateNumberQueryParam(req.query.size, 10);
   const { sort } = req.query;
+  const category = req.query.tags;
```

Comment:
get `tags` with `req.query` destructuring and if you want to rename it

```suggestion
const { sort, tags: category } = req.query
```

Reply:
I applied the suggested changes.